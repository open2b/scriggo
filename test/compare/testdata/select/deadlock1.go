// paniccheck

package main

func main() {
	ch := make(chan int)
	select {
	case <-ch:
	}
}
