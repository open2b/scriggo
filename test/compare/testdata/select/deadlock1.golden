fatal error: all goroutines are asleep - deadlock!